docs/website/docs/developers/debugging/sanitizers.md

```diff
@@ -42,11 +42,15 @@ cmake -DIREE_ENABLE_ASAN=ON ...
 #### Linking to the dynamic ASan runtime
 
 You may want to use ASan when using the python bindings.
-One way to achieve this is to build Python with ASan or whatever executable is
-going to use IREE as a shared library.
+One way to achieve this is to build Python (or whatever executable that is
+going to use IREE as a shared library) with Asan.
 Another option is to link to the ASan runtime dynamically instead of
 linking it statically into an executable.
 
+Configure IREE with something like.
```

Comment:
Using `clang-12` (other versions should also work) as a example, configure IREE with something like:

Reply:
I change it to your suggestion.